docs/content/docs/deployment/filesystems/gcs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Google cloud storage
+weight: 3
+type: docs
+aliases:
+  - /deployment/filesystems/gcs.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Google Cloud Storage
+
+[Google Cloud storage](https://cloud.google.com/storage) (GCS) provides cloud storage for variety of usecases. You can use it for **reading** and **writing data**.
```

Comment:
By its exclusion, this seems to suggest that GCS cannot be used for checkpointing. Is that the case?

```suggestion
[Google Cloud storage](https://cloud.google.com/storage) (GCS) provides cloud storage for a variety of use cases. You can use it for **reading** and **writing data**.
```